doc/nrf/ug_gzll.rst

```diff
@@ -401,39 +425,39 @@ The default channel tables require adjustment.
 
 Depending on your project, do one of the following:
 
-* Edit the :file:`gzll_params.h` file used in the nRF24Lxx projects.
-* Use the :c:func:`nrf_gzll_set_channel_table()` function in the nRF5 projects.
+* Edit the :file:`gzll_params.h` file used in the nRF24L Series projects.
+* Use the :c:func:`nrf_gzll_set_channel_table()` function in the nRF52 Series projects.
 
 Timeslot periods
 ================
 
 The Gazell Link Layer supports the following minimum timeslot periods:
 
-* 600 us timeslot period, nRF5 Gazell Device to nRF5 Gazell Host.
-* 504 us timeslot period, nRF5 Gazell Device to nRF24Lxx Gazell Host.
+* 600 us timeslot period, nRF52 Series Gazell Device to nRF52 Series Gazell Host.
+* 504 us timeslot period, nRF52 Series Gazell Device to nRF24L Series Gazell Host.
 
 When using 504 us timeslot period, the following restrictions apply:
 
 * The maximum payload size is 17 bytes.
-* The maximum ack payload size is 10 bytes.
+* The maximum ACK payload size is 10 bytes.
 
 In addition, the relation between the Device and Host timing parameters should be as follows:
 
-* The Host listens to each channel in a GZLL_RX_PERIOD number of microseconds, where GZLL_RX_PERIOD is the heartbeat interval in the nRF24Lxx devices.
+* The Host listens to each channel in a GZLL_RX_PERIOD number of microseconds, where GZLL_RX_PERIOD is the heartbeat interval in the nRF24L Series devices.
 * The Host GZLL_RX_PERIOD must be greater than the time required to make two full transmission attempts on the Device (including ACK wait time).
 
 Depending on your project, do one of the following:
 
-* Edit the :file:`gzll_params.h` file used in the nRF24Lxx projects.
-* Use the :c:func:`nrf_gzll_set_timeslot_period()` function in the nRF5 projects (nRF5 Gazell timeslot period = 0.5*GZLL_RX_PERIOD).
+* Edit the :file:`gzll_params.h` file used in the nRF24L Series projects.
+* Use the :c:func:`nrf_gzll_set_timeslot_period()` function in the nRF52 Series projects (nRF52 Series Gazell timeslot period = 0.5*GZLL_RX_PERIOD).
 
 Emulating legacy Gazell modes
 =============================
 
-The Gazell Link Layer protocol for the nRF5 Series is compatible with the most useful modes of the Gazell Link Layer for the nRF24Lxx devices.
+The Gazell Link Layer protocol for the nRF52 Series is compatible with the most useful modes of the Gazell Link Layer for the nRF24L Series devices.
 
-Emulating legacy nRF24Lxx Gazell Device mode 2 and nRF24Lxx Host mode 0
------------------------------------------------------------------------
+Emulating legacy nRF24L Series Gazell Device mode 2 and nRF24L Series Host mode 0
```

Comment:
Do we need to repeat nRF24L Series? The heading is already long without.

Reply:
That is a good point. I will remove the device name/ or re-write so that it is only written once.